Library/Homebrew/cli/args.rb

```diff
@@ -96,6 +98,19 @@ def resolved_formulae
         end.uniq(&:name).freeze
       end
 
+      def resolved_formulae_and_unknowns
+        return @resolved_formulae_and_unknowns if @resolved_formulae_and_unknowns
+
```

Comment:
```suggestion
        @resolved_formulae_and_unknowns ||= begin
```

and just the code below.